src/shared/components/Tooltip/Tooltip.style.ts

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled'
+import { colors, sizes, typography, zIndex, transitions } from '@/shared/theme'
+
+type TooltipProps = {
+  isActive?: boolean
+  arrowDisabled?: boolean
+}
+
+export const StyledTooltip = styled.div<TooltipProps>`
+  position: relative;
+  width: fit-content;
+  &:before {
```

Comment:
Let's use `::before` (with double colon), that's the newer syntax for pseudo-elements